indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -313,7 +313,14 @@ public static void injectSystemProperties(Job job)
   public static void injectDruidProperties(Configuration configuration, List<String> listOfAllowedPrefix)
   {
     String mapJavaOpts = configuration.get(MRJobConfig.MAP_JAVA_OPTS);
+    if (mapJavaOpts == null) {
+      mapJavaOpts = "";
+    }
+
     String reduceJavaOpts = configuration.get(MRJobConfig.REDUCE_JAVA_OPTS);
+    if (reduceJavaOpts == null) {
```

Comment:
Hi, Leventov. Thank you! I have used Strings.nullToEmpty according to your advice in #4588 based on master branch.